### PR TITLE
QA-Test tkt-49873 Adding timeout 10 minutes for update and jail api test

### PR DESCRIPTION
**Pull request by ericbsd**
If the time is out The test will fail and skip all test related to that test.

### CODE REVIEW

File: tests/api2/update.py [hunk 1]
```diff
@@ -90,6 +101,8 @@ def test_07_install_update():
 def test_08_verify_the_update_is_successful():
     if upgrade is False:
         pytest.skip('No update found')
+    elif download_hang is True:
+        pytest.skip('Download fail')
```
Comment (skarekrow) [inline tests/api2/update.py]: This should be more descriptive, *which* download. In this case getting the trains failed.

File: tests/api2/jail.py [hunk 2]
```diff
@@ -55,141 +56,207 @@ def test_04_fetch_bsd_release():
...
-            ]
-        }
-    )
-
-    assert results.status_code == 200, results.text
-
-    JOB_ID = results.json()
+    if freeze is True:
+        pytest.skip(freeze_msg)
+    else:
```
Comment (skarekrow) [inline tests/api2/jail.py]: Do we need all the else's in all these other files? If we say:
```
    if freeze is True:
         pytest.skip(freeze_msg)
```
Isn't that enough for it to skip? Less indented code is easier for readability.

Comment (ericbsd) [reply tests/api2/jail.py]: Yes, or they will fail.

File: tests/api2/update.py (hunk 1 above)
Comment (ericbsd) [reply tests/api2/update.py]: True

File: tests/api2/jail.py (hunk 2 above)
Comment (ericbsd) [reply tests/api2/jail.py]: Basically to avoid tests that depend on the 3 Job that can fail.

Comment (skarekrow) [reply tests/api2/jail.py]: I'm saying pytest wouldn't run the rest of that block would it? Then we can skip a lot of indented code.

**Pull request merged**